tentacoli.js

```diff
@@ -291,6 +294,21 @@ Tentacoli.prototype.request = function (data, callback) {
   return this
 }
 
+Tentacoli.prototype.fire = function (data, callback) {
+  var that = this
+  var req = new Request(null, callback)
+
+  try {
+    wrapStreams(that, data, req, true)
+  } catch (err) {
+    return this
```

Comment:
I'm ok with that. Pass the callback to `writeToStream` as well, in that case it can be used for control flow and backpressure.